index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="style">
+          <div class="right">
+           <h2>9:41</h2>
+          </div>
+          <div class="left">
+           <img src="/images/Cellular Connection.png" alt="">
+           <img src="/images/Wifi.png" alt="">
+           <img src="/images/Battery.png" alt="">
+          </div>
+        </div>
+        <nav class="nav">
+         <h1>My logo</h1>
+        <div class="menu">
+          <img src="/images/Union1.png" alt="">
+        </div>
+        </nav>
+    </header>
+    <div class="header-desc">
+        <h2>Hey There.
+        I’m Antoh.</h2>
+        <h3>I'm a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+        <div class="social">
+           <img src="/images/github (1).svg" alt="">
+           <img src="/images/Linkedin icon.svg" alt="">
+           <img src="/images/angel list.svg" alt="">
+           <img src="/images/twitter.svg" alt="">
+           <img src="/images/medium.svg" alt="">
+        </div>
```

Comment:
- [ ] I would kindly suggest you to consider use a `<section>` The container of the section is a `section` tag.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. please fix this

Reply:
- [ ] As mentioned by the previous reviewer, kindly wrap the "Headline" section in a `section` tag. It represents a generic standalone section of you portfolio, so it is good idea to use a `section` tag for semantic reasons. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.